hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/net/NetworkTopology.java

```diff
@@ -101,6 +101,13 @@ protected NetworkTopology init(InnerNode.Factory factory) {
   private int depthOfAllLeaves = -1;
   /** rack counter */
   protected int numOfRacks = 0;
+  /** empty rack map, rackname->nodenumber. */
+  private HashMap<String, Set<String>> emptyRackMap =
```

Comment:
This is a rack map to calculate the number of empty racks. So I prefer to use the name `rackMap`.
```suggestion
  private HashMap<String, Set<String>> rackMap =
```